docs/_index.md

```diff
@@ -13,7 +13,7 @@ without giving up JSON's uncanny ability to represent eclectic data.
 ## Getting Started
 
 Trying out SuperDB is easy: just [install](install.md) the command-line tool
-[`super`](commands/super.md) and run through the [tutorial](tutorials/zq.md).
+[`super`](commands/super.md) and run through its [usage documentation](commands/super.md).
```

Comment:
We talked in a team meeting recently about how the doc formerly known as the "zq tutorial" now reads more like a "`super` guide for `jq` users". It seems like it'll be good to keep that on in some form, but it's no longer the best starting point for new users. Meanwhile, @mccanne already overhauled the `super` command doc with lots of SQL-centric content, perf comparisons, and other goodies, so it's clearly a superior choice to link to.